src/scss/components/_button.scss

```diff
@@ -49,6 +49,15 @@ $button-padding-h: $grid-margin-2;
     border-right-color: $color-border !important;
     border-bottom-color: $color-border !important;
 
+    &:focus {
+        // TODO: can be removed once there are utrecht design tokens for the bottom/right
+        // border color
+        border-right-color: var(--of-color-info) !important;
+        border-bottom-color: var(--of-color-info) !important;
+        // TODO: can be removed once there are utrecht design tokens for the border-width
```

Comment:
Created [1406](https://github.com/nl-design-system/utrecht/issues/1406) for this - please add this issue reference in the comment.